cmd/peek-submission/expected_output_test.go

```diff
@@ -151,3 +151,7 @@ const incorrectSubmission = `{
     "submission-result": null
 }
 `
+
+const incorrectCourse = `{"message":"Could not find course: 'ZZZ'."}`
+
+const incorrectAssignment = `{"message":"Could not find assignment: 'zzz'."}`
```

Comment:
Don't put such small test case responses in a file.
We put the big ones here because they would make the actual test hard to read,
but that adds overhead.
For small strings, it's not worth the overhead.
Depending on the size of your output, you may want to break your test case over multiple lines (which is super common).